backend/tests/unit/api/test_chat_tools.py

```
@@ -499,17 +532,24 @@ async def test_tool_execution_error_returned_to_llm(self, test_client):
+                "backend.usecases.chat.chat_usecase.ToolRegistry"
+            ) as mock_registry_class,
         ):
             mock_llm_instance = MagicMock()
             mock_llm_instance.chat = AsyncMock(
                 side_effect=[tool_call_response, error_response]
             )
             mock_llm_class.return_value = mock_llm_instance
 
-            mock_conn = MagicMock()
-            mock_get_db.return_value = mock_conn
+            # ToolRegistryのモック（エラーを返す）
```

Ruff (0.14.11)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)